src/openforms/config/migrations/0031_move_analytics_tools_data.py

```diff
@@ -0,0 +1,154 @@
+# Generated by Django 3.2.15 on 2022-08-11 09:29
+
+from django.conf import settings
+from django.db import migrations
+
+from openforms.analytics_tools.constants import AnalyticsTools
+from openforms.analytics_tools.utils import (
+    get_cookies,
+    get_csp,
+    get_domain_hash,
+    update_analytical_cookies,
+    update_csp,
+)
+
+
+class Migration(migrations.Migration):
+    def move_analytics_config(apps, _):
+
+        AnalyticsToolsConfiguration = apps.get_model(
+            "analytics_tools", "AnalyticsToolsConfiguration"
+        )
+        GlobalConfiguration = apps.get_model("config", "GlobalConfiguration")
+        CookieGroup = apps.get_model("cookie_consent", "CookieGroup")
+
+        analytics_config = AnalyticsToolsConfiguration.objects.first()
+        config = GlobalConfiguration.objects.first()
+
+        if analytics_config and config:
+
+            if config.analytics_cookie_consent_group:
+                analytics_config.analytics_cookie_consent_group = (
+                    CookieGroup.objects.get(pk=config.analytics_cookie_consent_group.pk)
+                )
```

Comment:
```suggestion
            analytics_config.analytics_cookie_consent_group = config.analytics_cookie_consent_group
```

* if it's not set, on the `config`, then it'll just be `None` being assigned, which is fine
* if it exists, you can just directly set the instance - no need to re-query them, you already have the model instance.